File: 0027-remove-element/0027-remove-element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        
        indexer = 0
        
        for n in range(len(nums)):
            if nums[n] != val:
                nums[indexer] = nums[n]
                indexer += 1
                
        return indexer

File: 0027-remove-element/0027-remove-element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        Integer index = 0;
        for(int i = 0; i < nums.length; i++){
            if(nums[i] != val){
                nums[index] = nums[i];
                
                index += 1;
            }
        }
        return index;
    }
}